### DISCUSSION TITLE
How do you create a new branch and switch to it

### DISCUSSION THREAD

**Discussion by HakeemY** [community/community#135141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How do you create a new branch and switch to it

**Comment by Hakeemy2k** [comment]
Respond 1:

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/135143#discussioncomment-10277391) for reference to why this discussion was closed and locked. Thanks!
